addons/portal/migrations/9.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    openupgrade.rename_xmlids(cr, [
```

Comment:
As they are noupdate="0" records (all CSV records are this mode), and these records are not assigned manually to anything, you don't need to rename them, so put it as "Nothing to do" instead and remove this script.